packages/wrangler/src/api/startDevWorker/MultiworkerRuntimeController.ts

```diff
@@ -291,7 +291,7 @@ export class MultiworkerRuntimeController extends LocalRuntimeController {
 
 	#teardown = async (): Promise<void> => {
 		logger.debug("MultiworkerRuntimeController teardown beginning...");
-
+		this.cleanupContainers();
```

Comment:
Is this needed since the public `teardown()` method calls super, which will run the cleanup already?